prac_05/kivy_demo.py

```diff
@@ -0,0 +1,38 @@
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.properties import StringProperty
+from kivy.uix.button import Button
+
+
+class KivyDemo(App):
+    """Kivy program to demo some basic interactive functionality."""
+    status_text = StringProperty("Hello. Click the buttons :)")
+
+    def __init__(self, **kwargs):
```

Comment:
The docstring could be more descriptive, mentioning the initialization of counter and names.